smoke-tests/apps/Logback/src/smokeTest/java/com/microsoft/applicationinsights/smoketest/LogbackTest.java

```diff
@@ -101,9 +101,11 @@ void test() throws Exception {
       assertThat(md2.getProperties()).hasSize(3);
     }
 
-    if (!isWildflyServer()) {
-      assertThat(md3.getProperties()).containsEntry("Marker", "aMarker");
-    }
+    // TODO not sure why this entry no longers available.. can't find upstream changes related to
```

Comment:
@trask do you know any upstream changes related to this?

Reply:
ya, upstream changed marker attribute from string to string array: https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8074/files